src/Features/Traits/HttpResponse.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit\Framework\Assert;
+use Symfony\Component\CssSelector\CssSelectorConverter;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Response;
+
+trait HttpResponse
+{
+    protected Response $response;
+
+    /**
+     * @When a response is received
```

Comment:
I think I prefer "@when a request is sent". It's describing an action that is performed under the hood.